exercises/05/after/Messages/OrderBilled.cs

```diff
@@ -1,9 +1,8 @@
 ﻿using NServiceBus;
 
-namespace Messages
+namespace Messages.Events
 {
     public class OrderBilled
-        : IEvent
```

Comment:
Shouldnt this stay or is this using unobtrusive?

Reply:
This exercise moves from using the marker interfaces to unobstrusive, which is why the namespace changed